server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1361,13 +1361,6 @@ private void closeConsistencyCheck() {
       var tabletMeta = context.getAmple().readTablet(extent, ColumnType.FILES, ColumnType.LOGS,
           ColumnType.ECOMP, ColumnType.PREV_ROW);
 
-      if (!tabletMeta.getExtent().equals(extent)) {
```

Comment:
Could keep this and change the check to `if(tabletMetadata == null)` and the error message to `"Closed tablet" + extent + " does not exists in metadata"`

Reply:
Added this in 946b7bb